deepspeed/runtime/zero/partition_parameters.py

```diff
@@ -479,8 +479,11 @@ def get_model():
 
         self._validate_remote_device(remote_device, _ds_config)
 
+        self.avaliable_parameter_numel = 0
```

Comment:
```suggestion
        self.available_parameter_numel = 0
```

and everywhere else :)